xtask/src/main.rs

```diff
@@ -170,8 +166,12 @@ Usage: Run with `cargo xtask <task>`, eg. `cargo xtask docgen`.
     Tasks:
         docgen: Generate files to be included in the mdbook output.
         query-check: Check that tree-sitter queries are valid.
+<<<<<<< HEAD
         code-gen: Generate files associated with steel
         steel: Install steel
+=======
+        theme-check: Check that theme files in runtime/themes are valid.
+>>>>>>> origin
```

Comment:
Just want to note that some merge-conflicts sneaked in here (as well for commented-out code in `helix-term/src/commands/typed.rs`)